activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/channel/ServiceTaskIntegrationResultEventHandler.java

```diff
@@ -48,7 +48,7 @@ public synchronized void receive(IntegrationResultEvent integrationResultEvent)
         IntegrationContextEntity integrationContext = integrationContextService.findIntegrationContextByExecutionId(integrationResultEvent.getExecutionId());
 
         if (integrationContext != null) {
-
+            integrationContextService.deleteIntegrationContext(integrationContext);
```

Comment:
@erdemedeiros We need a test that removes the execution before this event arrives

Reply:
Yeah, I see, I are thinking about a terminate event or boundary event behaviour, or similar. The current fix should definitely be updated